tools/lsp/preview/element_selection.rs

```diff
@@ -27,6 +27,18 @@ impl ElementRcNode {
         Some(Self { element, debug_index })
     }
 
+    pub fn on_element_debug<R>(
```

Comment:
i think the convention is rather to use the `with_` prefix rather than `on_`,  the later being used for callback that happens when something change.